fs/sync/sync.go

```diff
@@ -20,6 +20,14 @@ import (
 	"github.com/rclone/rclone/fs/operations"
 )
 
+// errorMaxDurationReached defines error when transfer duration is reached
+// Used for checking on exit and matching to correct exit code.
+var errorMaxDurationReached = errors.New("max transfer duration reached as set by --max-duration")
```

Comment:
Make the name `ErrorMaxDurationReached` and it will be exported. You'll need to change the comment above too.

Reply:
sorry for that - it was capitalized. but I manually copied changes from one branch to another and used wrong clip book.

Anyway - now it is corrected.

Reply:
You also need to import `fs/sync` so a import like this:

`fssync "github.com/rclone/rclone/fs/sync"` (fssync so that it doesn't collide with the golang sync package).

Reply:
"fssync" was what I was missing - I could not figure out how to import it.

Thank you very much for help. It is already working and throwing exit code 10:)

Now a bit of tests and last touches and should be ready for review.